jlm/util/Statistics.hpp

```diff
@@ -339,12 +339,15 @@ class StatisticsCollectorSettings final
 
   /**
    * @return the directory used for outputting statistics and debug output files.
-   * If no output directory is given, an assertion failure occurs.
+   *
+   * \note If no output directory path is given, an assertion failure occurs. If the directory
+   * does not exist yet, it is created.
    */
   [[nodiscard]] const FilePath &
-  GetOutputDirectory() const noexcept
+  GetOrCreateOutputDirectory() const noexcept
```

Comment:
I think it might still make sense to keep a pure getter, if only to handle the comment I wrote above

Reply:
Added it back.